src/lib/y2partitioner/widgets/pages/btrfs_filesystems.rb

```diff
@@ -45,7 +45,7 @@ def initialize(filesystems, pager)
 
         # @macro seeAbstractWidget
         def label
-          Y2Storage::Filesystems::Type::BTRFS.to_human
+          _("Btrfs")
```

Comment:
Why this change?

Reply:
Well, originally it was as it is now, but I changed it in my previous PR. But now, I have realized that maybe we need such Btrfs label in different places, so maybe it is better to have a translated string for it, instead of calling the long `Y2Storage::Filesystems::Type::BTRFS.to_human` everywhere.